### ISSUE TITLE
Receiver Plane Depth Bias has problems on iPhone 6/6S

### ISSUE THREAD

**Issue by willeastcott** [playcanvas/engine#722]
Check out this project:

https://playcanvas.com/project/419882/overview/jp-shadow-bug

On iPhone 5S:
![slack for ios upload 3](https://cloud.githubusercontent.com/assets/697563/18165296/4e8dccae-703c-11e6-8eb7-eb5ad14f058b.jpg)

On iPhone 6/6S:
![slack for ios upload 4](https://cloud.githubusercontent.com/assets/697563/18165299/57a8feb2-703c-11e6-8d58-554047c527f5.jpg)

Doing the following fixes the problem (effectively disabling RPDB):

```
// ...in initialize function
this.app.graphicsDevice.extStandardDerivatives = null;
```

**Comment by guycalledfrank** [comment]
Receiver Plane Depth Bias is removed on WebGL 2.0. Will try to fix 1.0 too soon.

**Comment by willeastcott** [comment]
Closing due to:

1. Age
2. The original linked repro project has since been deleted

Feel free to reopen if a new repro can be created.
